### ISSUE TITLE
Hide input-selection context menu on mobile

### ISSUE THREAD

**Issue by kb-tl** [loebi-ch/jquery-clock-timepicker#22]
Is it possible to hide the context menu (select all/copy/paste etc) on mobile that appears when inputs have text selected?

![2019-01-28 14 07 27](https://user-images.githubusercontent.com/5319854/51841305-36f03a80-2306-11e9-9eee-2da36044bf6f.jpg)

**Comment by loebi-ch** [comment]
What Mobile Phone? Which browser?

I cannot reproduce on Android Chrome browser...

**Comment by kb-tl** [comment]
Sorry.... this is on the default Samsung browser on the Galaxy S8.

**Comment by reclaimingmytime** [comment]
I can reproduce the issue when using the "Samsung Internet Browser" (manually downloaded from the Play Store) on my OnePlus 5.

The numbers get selected as soon as the timepicker opens. This issue seems to be related to how "Samsung Internet" behaves when focusing an element using JavaScript.

**Comment by loebi-ch** [comment]
Thank you @reclaimingmytime!
I could reproduce the issue as well on Samsung browser... But I haven't found any solution by googling it. It will need some coding experiments, but currently I don't find the time therefore...

Does anyone have an idea what it could be and how to solve it? Please contribute 😉

**Comment by reclaimingmytime** [comment]
The context menu seems to be caused by `selectHourOnInputElement` and `selectMinuteOnInputElement`:
```js
function selectHourOnInputElement() {
    inputElement.focus();
    setTimeout(function() {
        inputElement.get(0).setSelectionRange(0, inputElement.val().indexOf(settings.separator));
    }, 1);
}
```
When commenting out the code inside this function, the hours are no longer selected, but the context menu disappears.

I could reproduce the issue in a simple jsFiddle that selects the contents of an input field when clicking on it: https://jsfiddle.net/xrh1a3ng/2/

Only Samsung Internet shows the context menu. That browser tries to be helpful by offering the context menu.

It seems like the issue is that the context menu appears automatically when **selecting** text. A possible solution could be offering a similar visual representation without actually selecting the hours/minutes.

**Comment by loebi-ch** [comment]
Fixed with version 2.3.2
